app/models/current_user_data.rb

```diff
@@ -0,0 +1,43 @@
+CurrentUserData = Struct.new(:user, :restricted) do
+  def data
+    serializer.new(user, scope: serializer_scope, root: :current_user).as_json.tap do |json|
+      json[:current_user].except!(:group_ids, :membership_ids)
```

Comment:
It interferes with our `groupIds()` method on the frontend if these get serialized out. (Normally when you use `has_many` with ActiveModel::Serializers, it includes the ids of the child record with the parent. We don't actually care about that though.)